roles/matrix-nginx-proxy/templates/systemd/matrix-nginx-proxy.service.j2

```diff
@@ -22,6 +22,9 @@ ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-nginx-proxy \
 			--cap-drop=ALL \
 			--read-only \
 			--tmpfs=/tmp:rw,noexec,nosuid,size={{ matrix_nginx_proxy_tmp_directory_size_mb }}m \
+			{% if matrix_nginx_proxy_enabled and matrix_nginx_proxy_synapse_cache_enabled %}
```

Comment:
I believe that the `matrix-nginx-proxy.service` file is only created if `matrix-nginx-proxy` is enabled (`matrix_nginx_proxy_enabled`), so adding this in the `if` statement is superfluous.

We should leave `if matrix_nginx_proxy_synapse_cache_enabled` only.

Which makes me think.. if this cache is just for Synapse, and we're calling the variable and cache zone name "synapse", we should probably make the directory `/tmp/synapse-cache` instead?

Reply:
You are right. Changed it accordingly